src/components/testimonialCard/testimonial-card.css

```diff
@@ -0,0 +1,56 @@
+.testimonial-container{
+  background-color: rgb(71, 79, 197);
+  height: 100%;
+  border-radius: 1rem;
+  position: relative;
+}
+
+.testimonial-container__purple{
+  background-color: rgb(150, 0, 236);
+  color: white;
+}
+
+.testimonial-container__grey{
+  background-color: rgb(155, 155, 155);
+  color: white;
+}
+
+.testimonial-container__white{
+  background-color: white;
+}
+
+.testimonial-container__dark-grey{
+  background-color: rgb(56, 56, 56);
+  color: white;
+}
+
+.testimonial__header--name{
```

Comment:
Same with the BEM naming convention, rather do something like:
`.testimonial-header__name`
i.e. start a new block with `.testimonial-header` and create only one level of children underneath it.